to-headers/index.js

```diff
@@ -22,3 +22,5 @@ module.exports = (headers) => {
 
     return headers;
 };
+
+const supportsHeaders = () => typeof Headers === 'function';
```

Comment:
how can you tell that global.Headers is the constructor you expect?
maybe you should import the relevant Headers if it's not exists?

Reply:
If someone polyfills Headers badly, it's really their problem